html_css/project_flexbox.md

```diff
@@ -0,0 +1,19 @@
+### Project: Tribute Page
+In this project, you'll get to take everything you've learned about flex box and use it to build your own simple tribute page As with the other projects, we'll keep it relatively open-ended to give you a chance to exercise your creativity and problem-solving skills. It will be a great project to show off to potential employers as well.
+
+Once you have learned Flex box which is (aka Flexible Box) you know that the Flexbox is a pretty new concept in CSS. It solves a lot of problems of Web design like the Holy Grail Layout. Another area where Flexbox helps is appending and prepending elements to other elements.you know that it's super challenging to create a responsive web page.But with Flexbox it’s super simple.
```

Comment:
whats the holy grail layout?

Reply:
![alt text](https://upload.wikimedia.org/wikipedia/commons/thumb/a/ad/HolyGrail.svg/220px-HolyGrail.svg.png "Holy Grail Layout")
This is called "Holy Grail Layout". @KevinMulhern